build.gradle

```diff
@@ -125,7 +125,25 @@ task testVersionCommand(dependsOn: jar) {
 	}
 }
 
-check.dependsOn testExecutableJar, testVersionCommand
+task testHelpCommand(dependsOn: jar) {
```

Comment:
For help commands lets make normal unit tests.  These tests don't need the phyical jar to be created like the version and exe tests do.  So lets convert them to normal unit tests